### PR TITLE
IHT-4877 ApplicationKickOutHelper refactoring

### PR DESCRIPTION
**Pull request by pankaj2453**

### CODE REVIEW

File: app/iht/utils/ApplicationKickOutHelper.scala [hunk 1]
```diff
@@ -16,21 +16,12 @@
 
 package iht.utils
 
-import iht.constants.IhtProperties
```
Comment (tyagikumarvineet) [inline app/iht/utils/ApplicationKickOutHelper.scala]: Make the below line shorter as its too long
 def returnLinkUrl(kickoutReason: String, ihtRef: String, applicationLastSection: Option[String], applicationLastID: Option[String]): Option[Call] = {

*Might refactor by having parameters on the seprate line

File: test/iht/utils/ApplicationKickoutCommonHelperTest.scala [hunk 2]
```diff
@@ -24,10 +24,9 @@ import iht.models.application.exemptions.BasicExemptionElement
 import iht.testhelpers.CommonBuilder
```
Comment (tyagikumarvineet) [inline test/iht/utils/ApplicationKickoutCommonHelperTest.scala]: As discussed, rename it to ApplicationKickOutNonSummaryHelper

File: app/iht/utils/ApplicationKickOutHelper.scala (hunk 1 above)
Comment (pankaj2453) [inline app/iht/utils/ApplicationKickOutHelper.scala]: Implemented

File: test/iht/utils/ApplicationKickoutCommonHelperTest.scala (hunk 2 above)
Comment (pankaj2453) [inline test/iht/utils/ApplicationKickoutCommonHelperTest.scala]: Implemented

**Pull request merged**